pkg/kubelet/kubelet_pods.go

```diff
@@ -1974,6 +1974,13 @@ func (kl *Kubelet) convertToAPIContainerStatuses(pod *v1.Pod, podStatus *kubecon
 			} else {
 				determineResource(v1.ResourceCPU, container.Resources.Limits, oldStatus.Resources.Limits, limits)
 			}
+			// 10m is the minimum cpu limit
+			// so setting it back the limit to user provided value to avoid confusion for user
+			// More info: https://github.com/kubernetes/kubernetes/pull/102884#discussion_r662552642
+			if container.Resources.Limits.Cpu().MilliValue() <= 10 {
```

Comment:
is there not a need for `container.Resource.Limits != nil` condition checked like what you're implementing with `container.Resources.Requests`?

Reply:
Its already checked in line no: 1970, Even additional check in request is also need to be removed.

Reply:
I see. I agree. Thank you for the clarification.